src/annotator/sidebar.js

```diff
@@ -63,9 +88,9 @@ export class Sidebar {
    * @param {HTMLElement} element
    * @param {import('./util/emitter').EventBus} eventBus -
    *   Enables communication between components sharing the same eventBus
-   * @param {Record<string, any>} [config]
+   * @param {SidebarContainerConfig & SidebarConfig} config
    */
-  constructor(element, eventBus, config = {}) {
+  constructor(element, eventBus, config) {
```

Comment:
The default `{}` argument was removed because it is not used and is missing the required `sidebarAppUrl` property.